src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,768 @@
+
+	"jei.securitycraft:reinforcedBlock.info": "Haz clic derecho con un reforzador de bloques universal y coloca el siguiente bloque en el hueco. Si lo deseas, también puedes colocar el bloque y romperlo con el reforzador de bloques universal.",
+
+	"config.waila.plugin_securitycraft": "SecurityCraft",
+	"config.waila.plugin_securitycraft.showowner": "¿Mostrar propietario?",
+	"config.waila.plugin_securitycraft.showmodules": "¿Mostrar módulos?",
+	"config.waila.plugin_securitycraft.showpasswords": "¿Mostrar contraseñas?",
+	"config.waila.plugin_securitycraft.showcustomname": "¿Mostrar nombre de bloques?",
+	"waila.securitycraft:owner": "Propietario:",
+	"waila.securitycraft:equipped": "Equipado con:",
+	"waila.securitycraft:password": "Contraseña:",
+	"waila.securitycraft:password.notSet": "Sin establecer",
+	"waila.securitycraft:customName": "Nombre personalizado:",
+	"waila.securitycraft:customName.notSet": "Sin establecer",
+
+	"tooltip.securitycraft:mine": "Mina",
+	"tooltip.securitycraft:sentry": "Torreta",
+	"tooltip.securitycraft:camera": "Cámara",
+	"tooltip.securitycraft:cameraMonitor": "# cámaras vinculadas:",
+	"tooltip.securitycraft:module.modifiable": "¡Módulo modificable!",
+	"tooltip.securitycraft:module.notModifiable": "¡Módulo no modificable!",
+	"tooltip.securitycraft:module.playerCustomization.players": "Jugadores",
+	"tooltip.securitycraft:module.itemAddons.usage.blocks": "Puedes añadir # bloque(s).",
+	"tooltip.securitycraft:module.itemAddons.usage.items": "Puedes añadir # objeto(s).",
+	"tooltip.securitycraft:module.itemAddons.usage.blocksAndItems": "Puedes añadir # bloque(s) y $ objeto(s).",
+	"tooltip.securitycraft:module.itemAddons.added": "Añadido",
+	"tooltip.securitycraft:keycard.uses": "Usos restantes:",
+
+	"itemGroup.securitycraft.decoration": "SecurityCraft: Decoración",
+	"itemGroup.securitycraft.explosives": "SecurityCraft: Explosivos",
+	"itemGroup.securitycraft.technical": "SecurityCraft: Técnico",
+
+	"module.securitycraft.ims.whitelist_module.description": "Al añadir un módulo de lista blanca a un S.M.I., permitirá que dichos jugadores se acerquen sin que les ataque.",
+	"module.securitycraft.inventory_scanner.whitelist_module.description": "Al añadir un módulo de lista blanca a un escáner de inventario, permitirá que dichos jugadores pasen por el escáner sin que se rastree su inventario.",
+	"module.securitycraft.inventory_scanner.smart_module.description": "Al añadir un módulo inteligente a un escáner de inventario, permitirá comprobar el inventario para encantamientos y otros cambios de metadatos.",
+	"module.securitycraft.inventory_scanner.storage_module.description": "Al añadir un módulo de almacenamiento a un escáner, permitirá que los escáneres almacenen los objetos en vez de eliminarlos del inventario.",
+	"module.securitycraft.inventory_scanner.disguise_module.description": "Al añadir un módulo de disfraz a un escáner de inventario, permitirá que cambie el modelo y la tex
```

Comment:
Fixing it! (I was wondering if this string must be interrogative... I submitted it without interrogation marks because it doesn't have in the original file, but the logic makes me think that must have)

Reply:
It's ok as it is, don't worry